package.json

```diff
@@ -27,7 +27,7 @@
     "deploy-nightly": "yarn build && node scripts/deploy_nightly_build.js"
   },
   "peerDependencies": {
-    "react": "^16.13.1"
+    "react": "^0.0.0-experimental-94c0244ba"
```

Comment:
i just realized that it'll probably change it for the outside world as well?
@davidmccabe @drarmstr testing this thing in concurrent mode requires the experimental build of react, is that what we run in WWW? 
i'm wondering if there's an easy way to patch the version in test environment